client/components/UI/List/Group.tsx

```diff
@@ -7,35 +7,44 @@ import classNames from 'classnames';
  * @name Group
  * @description Component to group a list of items
  */
-export const Group = ({
-    className,
-    children,
-    spaceBetween,
-    verticalScroll,
-    style,
-    onScroll,
-    refNode,
-    shadow,
-    testId,
-}) => (
-    <div
-        data-test-id={testId}
-        className={classNames(
-            className,
-            'sd-list-item-group',
-            shadow ? `sd-shadow--z${shadow}` : null,
-            {
-                'sd-list-item-group--space-between-items': spaceBetween,
-                'sd-list-item-group--vertical-scroll': verticalScroll,
-            }
-        )}
-        style={style}
-        onScroll={onScroll}
-        ref={refNode}
-    >
-        {children}
-    </div>
-);
+export const Group = (props) => {
```

Comment:
In my PR, there is now the `IProps` interface defined for this component:
https://github.com/superdesk/superdesk-planning/blob/99e16ac3c2529b744b72631e57a5f58b65d1c7c5/client/components/UI/List/Group.tsx